wbooks/reactotronConfig.js

```diff
@@ -0,0 +1,32 @@
+import Reactotron from 'reactotron-react-native';
+import { reactotronRedux } from 'reactotron-redux';
+
+const reactotronConfig = {
```

Comment:
Try using the same config as [here ](https://docs.google.com/document/d/1i_Dq2sW6CjePw40i_84Y1TVXle7he2q6CnUekkVDAZI/edit) because this one is breaking the tests.

Move `reactotronConfig.js` to a `config` folder.